opentelemetry/src/main/scala/zio/telemetry/opentelemetry/tracing/Tracing.scala

```diff
@@ -414,6 +414,12 @@ trait Tracing { self =>
     trace: Trace
   ): UIO[Span]
 
+  def setErrorStatus[E <: Throwable](
+    span: Span,
+    cause: Cause[E],
+    errorMapper: ErrorMapper[E]
+  )(implicit trace: Trace): UIO[Span]
+
```

Comment:
Why do you think we need to make it a part of the public API?

Reply:
I made it public as in OpenTracing
https://github.com/zio/zio-telemetry/blob/series/2.x/opentracing/src/main/scala/zio/telemetry/opentracing/OpenTracing.scala#L17

Reply:
I see. The thing is that opentracing module needs some love, which I'm planning to give it when we stabilize the API of opentelemetry, which is our playground for the moment. So please, do not refer to opentracing module as an example.

Considering the above I would like to ask you about making this method private. Thanks!

Reply:
fixed